tests/unit_tests/main.cc

```diff
@@ -31,4 +31,19 @@
 #define BOOST_TEST_MAIN
 #include <boost/test/unit_test.hpp>
 
+#include "core/util/log.h"
+
+/// @brief Disable log spam from the libs for all unit-tests
+/// @note This won't disable logs that boost.test specific
+struct Logger final
+{
+  Logger()
+  {
+    boost::log::core::get()->set_logging_enabled(false);
```

Comment:
This is awesome! Tried it out, and everything works great. If a user needs log messages in tests, they can use `BOOST_TEST_MESSAGE`, and run with `./build/kovri-tests --log_level=message`. LGTM.